src/store/modules/ResourceManagement/SystemParametersStore.js

```diff
@@ -69,138 +74,67 @@ const systemParametersStore = {
             immediateTestRequested[0].CurrentValue;
-          let modeValue =
+          let immediateTestRequestedModeValue =
             immediateTestRequestedValue == 'Enabled' ? true : false;
-          commit('setImmediateTestRequested', modeValue);
-        })
-        .catch((error) => console.log(error));
-    },
-    async getGuardOnError({ commit }) {
-      return await api
-        .get(
-          '/redfish/v1/Registries/BiosAttributeRegistry/BiosAttributeRegistry'
-        )
-        .then(({ data: { RegistryEntries } }) => {
-          const immediateTestRequested = RegistryEntries.Attributes.filter(
+          commit('setImmediateTestRequested', immediateTestRequestedModeValue);
+          const guardOnError = this.registryEntries.Attributes.filter(
             (Attribute) => Attribute.AttributeName == 'pvm_rpd_guard_policy'
           );
-          let immediateTestRequestedValue =
-            immediateTestRequested[0].CurrentValue;
-          let modeValue =
-            immediateTestRequestedValue == 'Enabled' ? true : false;
-          commit('setGuardOnError', modeValue);
-        })
-        .catch((error) => console.log(error));
-    },
-    async getRpdPolicyOptions({ commit }) {
-      return await api
-        .get(
-          '/redfish/v1/Registries/BiosAttributeRegistry/BiosAttributeRegistry'
-        )
-        .then(({ data: { RegistryEntries } }) => {
-          const rpdPolicy = RegistryEntries.Attributes.filter(
+          let guardOnErrorCurr = guardOnError[0].CurrentValue;
+          let guardOnErrorModeValue =
+            guardOnErrorCurr == 'Enabled' ? true : false;
+          commit('setGuardOnError', guardOnErrorModeValue);
+          const rpdPolicyOps = this.registryEntries.Attributes.filter(
             (Attribute) => Attribute.AttributeName == 'pvm_rpd_policy'
           );
-          let rpdPolicyOptions = rpdPolicy[0].Value.map(
+          let rpdPolicyOptions = rpdPolicyOps[0].Value.map(
             ({ ValueName }) => ValueName
           );
           commit('setRpdPolicyOptions', rpdPolicyOptions);
-        })
-        .catch((error) => console.log(error));
-    },
-    async getRpdFeatureOptions({ commit }) {
-      return await api
-        .get(
-          '/redfish/v1/Registries/BiosAttributeRegistry/BiosAttributeRegistry'
-        )
-        .then(({ data: { RegistryEntries } }) => {
-          const rpdPolicy = RegistryEntries.Attributes.filter(
+          const rpdFeatureOps = this.registryEntries.Attributes.filter(
             (Attribute) => Attribute.AttributeName == 'pvm_rpd_feature'
           );
-          let rpdFeatureOptions = rpdPolicy[0].Value.map(
+          let rpdFeatureOptions = rpdFeatureOps[0].Value.map(
             ({ ValueName }) => ValueName
           );
           commit('setRpdFeatureOptions', rpdFeatureOptions);
-        })
-        .catch((error) => console.log(error));
-    },
-    async getRpdScheduledRun({ commit }) {
```

Comment:
There is an error. The methods that you have removed, But being called somewhere in the code.

Reply:
Make sure that the method name that is removed is not being used anywhere in the code. Eg: Click on "Update RPD policy" and there is vuex error in console.
![image](https://github.com/ibm-openbmc/webui-vue/assets/96164786/e1486c96-b1dd-4e45-a6c2-6fa2e56a837a)